pkg/scheduler/api/devices/nvidia/vgpu/utils.go

```diff
@@ -390,17 +393,27 @@ func checkNodeGPUSharingPredicate(pod *v1.Pod, gssnap *GPUDevices, replicate boo
 					Usedmem:   val.Memreq,
 					Usedcores: val.Coresreq,
 				})
+				switch SchedulePolicy {
```

Comment:
How about abstract an interface here and call different score implement by different policy?

Reply:
it may be hard to design and abstract an device-compatable schedule policy interface, because the same strategy in one device can't be applied to other devices(vGPU, gpushare, npu, etc...) without modifying parameters